website/src/css/custom.less

```diff
@@ -203,7 +193,158 @@ article {
                 display: none;
             }
         }
+
+        p>code {
+            font-size: 13px;
+            padding: 3px 6px;
+            border-radius: 3px;
+            background-color: rgba(150, 150, 150, 0.2);
+        }
+    }
+}
+
+.small-table {
+    font-size: 12px;
+    margin-top: 20px;
+    margin-bottom: 30px;
+
+    td {
+        padding: 10px;
+    }
+}
+
+[class*="quickstart-solidity-hardhat"] {
```

Comment:
This ensures that the custom styling only applies to the quickstart. If we want to see these styles applied more consistently throughout the docs, we could easily do that (ideally via another PR).